lib/inbox/widgets/inbox_replies_view.dart

```diff
@@ -51,116 +53,128 @@ class _InboxRepliesViewState extends State<InboxRepliesView> {
       return Align(alignment: Alignment.topCenter, heightFactor: (MediaQuery.of(context).size.height / 27), child: const Text('No replies'));
     }
 
-    return ListView.builder(
-      shrinkWrap: true,
-      physics: const NeverScrollableScrollPhysics(),
-      itemCount: widget.replies.length,
-      itemBuilder: (context, index) {
-        return Column(
-          children: [
-            Divider(
-              height: 1.0,
-              thickness: 1.0,
-              color: ElevationOverlay.applySurfaceTint(
-                Theme.of(context).colorScheme.surface,
-                Theme.of(context).colorScheme.surfaceTint,
-                10,
+    return BlocListener<InboxBloc, InboxState>(
+      listener: (context, state) {
+        if (state.status == InboxStatus.success) {
+          setState(() {
```

Comment:
Should we invert this statement so that we don't set the state unecessarily?

```dart
if (inboxReplyMarkedAsRead == null) return;

setState(() {
  inboxRepliesMarkedAsRead.add(inboxReplyMarkedAsRead!);
  inboxReplyMarkedAsRead = null;
});

Reply:
Good idea! Done in c7a61503f7921b595aedf2c43234d8a0d3640b95.